automated_provisioning/README.md

```diff
@@ -124,6 +122,7 @@ The command should look something like the below (with the parameters replaced b
 ## Future work
 - Add more validation on input strings if issues come up.
```

Comment:
Suggestion: capture the existing and future work as issues on the repo. You can just "lift and shift" (line &mapsto; issue title; empty body).

Reply:
good call, done